src/components/Markdown.svelte

```diff
@@ -52,14 +52,10 @@
   }
 
   function render(content: string): string {
-    return dompurify.sanitize(
-      marked.parse(content, {
-        renderer: new Renderer(linkBaseUrl),
-        // TODO: Disables deprecated options, remove once removed from marked
-        mangle: false,
-        headerIds: false,
-      }),
-    );
+    markdown.setOptions({
+      renderer: new Renderer(linkBaseUrl),
+    });
+    return dompurify.sanitize(markdown.parse(content));
```

Comment:
I believe this modifies the options of the shared `Markdown` instance. This may mess with other places where we use `markdown`. I would suggest the following:

```suggestion
    return dompurify.sanitize(markdown.parse(content, {
      renderer: new Renderer(linkBaseUrl)
    }));
```

Reply:
Nice, yeah that seems better 👍

Reply:
Added